src/components/todoDone.js

```diff
@@ -0,0 +1,222 @@
+        margin:10px;
+        padding:10px;
+        border-radius:8px;
+        background-color:#D93535;
+        width:90%;
+        
+    }
+    .todoListItemXBtn{
+        position:absolute;
+        width:25px;
+        height:30px;
+        top:0px;
+        right:10px;
+        text-align:center;
+    }
+
+    .todoListItemWriteBtn{
+        position:absolute;
+        right:35px;
+        top:5px;
+        width:25px;
+        height:25px;
+        text-align:center;
+    }
+    .todoListEditContainer{
+        display:flex;
+        flex-direction:column;
+        position:relative;
+    }
+    .todoListEditTitle{
+        border-radius:8px;
+        background-color: rgb(200, 200, 200);
+        margin:2px;
+        font-size:20px;
+        width:50%;
+    }
+    .todoListEditContent{
+        border-radius:8px;
+        background-color: rgb(200, 200, 200);
+        margin:2px;
+    }
+    .todoListEditBtn{
+        border-radius:8px;
+        background-color: rgb(200, 200, 200);
+        margin:2px;
+
+    }
+    .hoverMoveTodoBtn{
+        display:none;
+        border-radius:8px;
+        background-color:white;
+    }
+    .hoverMoveProgressBtn{
+        border-radius:8px;
+        backgroudn-color:white;
+    }  
+    .hoverMoveDoneBtn{
+        border-radius:8px;
+        background-color:white;
+    }
+    .todoListRedBtn{
+        position:absolute;
+        border-radius:50%;
+        background-color:#D93535;
+        width:30px;
+        height:30px;
+        top:0px;
+        left:55%;
+        border:1px solid white;
+    }
+    .todoListBlueBtn{
+        position:absolute;
+        border-radius:50%;
+        background-color:#307FE2;
+        width:30px;
+        height:30px;
+        top:0px;
+        left:67%;
+        border:1px solid white;
+    } .todoListGreenBtn{
+        position:absolute;
+        border-radius:50%;
+        background-color:#00B294;
+        width:30px;
+        height:30px;
+        top:0px;
+        left:79%;
+        border:1px solid white;
+    } .todoListPurpleBtn{
+        position:absolute;
+        border-radius:50%;
+        background-color:#7678D1;
+        width:30px;
+        height:30px;
+        top:0px;
+        left:91%;
+        border:1px solid white;
+    }
+
+`
+
+function TodoDone(){
+    const [todos, setTodos] = useState([]);
+    const [todoDone, setDone] = useState([]);
+    const [todoProgress, setProgress] = useState([]);
+    useEffect(() => {
+        const savedTodos = JSON.parse(localStorage.getItem('todos')) || [];
+        const savedDone = JSON.parse(localStorage.getItem('todoDone')) || [];
+        const savedProgress = JSON.parse(localStorage.getItem('todoProgress')) || [];
+        if(savedDone.length>0){
+            setDone(savedDone)
+        }
+        if(savedProgress.length>0){
+            setProgress(savedProgress)
+        }
+        if(savedTodos.length>0){
+            setTodos(savedTodos);
+        }
+    }, [todoDone]);
```

Comment:
todoDone의 경우는 페이지가 렌더링 된 이후 해당 useEffect외부의 함수에 의해 변경되지 않아서
처음 페이지가 렌더링 될 때 한번만 useEffect가 작동을 하고 그 이후는 trigger되지 않을 것같습니다.
dependency array에 어떠한 내용이 들어가야할지 다시 한번 고민해보는것도 좋을 것 같습니다.

또한, todo, tododone, todoProgress의 초기값이 []이기 때문에  아래와 같이 적어도 될 것 같아요
```suggestion
    useEffect(() => {
       setTodos(JSON.parse(localStorage.getItem('todos')) || []);
       ...
    }, [?????]);
```